plugins/ohif/config/nginx.conf.template

```diff
@@ -35,14 +35,15 @@ http {
 
         location /monaiservice/ {
             proxy_http_version 1.1;
-            proxy_set_header   Host               $host;
+            proxy_set_header   Host               "${MONAI_SERVICE_URI}";
             proxy_set_header   X-Real-IP          $remote_addr;
             proxy_set_header   X-Forwarded-For    $proxy_add_x_forwarded_for;
             proxy_set_header   X-Forwarded-Proto  $scheme;
 
             expires            0;
             add_header         Cache-Control private;
-            proxy_pass         ${MONAI_SERVICE_URI};
+            proxy_pass         "https://${MONAI_SERVICE_URI}/api/v1/";
+
             proxy_buffering    off;
```

Comment:
instead of `nvidia.pem` installation etc.. you could have tried 
`proxy_ssl_verify              off;` here to ignore ssl verification